tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/CDITestBase.java

```diff
@@ -110,12 +160,20 @@ public void prepareWorkspace() {
 			cw.open();
 			WebProjectFirstPage fp = new WebProjectFirstPage(cw);
 			fp.setProjectName(PROJECT_NAME);
+			if (CDIVersion.equals("2.0")) {
+				new DefaultCombo(2).setSelection("Dynamic Web Project with CDI 2.0 (Contexts and Dependency Injection)");
+			}
 			fp.setTargetRuntime(sr.getRuntimeName());
-			fp.activateFacet("1.8", "Java");
+			if (isJavaLE8() || CDIVersion.equals("1.0")) {
+				fp.activateFacet("1.8", "Java");
+			} else {
+				fp.activateFacet("11", "Java");
+			}
 			fp.activateFacet(CDIVersion, "CDI (Contexts and Dependency Injection)");
 			cw.finish();
 			new WaitUntil(new JobIsRunning(), TimePeriod.DEFAULT, false);
 			new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+			setWorkspaceDefaultJREForJava8();
```

Comment:
Should not it be set by default when it is JRE 8 on the machine?

Reply:
Fixed.